Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -11689,6 +11797,40 @@ end
 					},
 					Notes = "Returns true if the specified player is queued to be transferred to a World.",
 				},
+				RegisterForgeMod =
+				{
+					Params =
+					{
+						{
+							Name = "ModName",
+							Type = "string",
+						},
+						{
+							Name = "ModVersion",
+							Type = "string",
+						},
+					},
+					Notes = "Add a Forge mod name/version to the server ping list.",
```

Comment:
What happens if two plugins register the same `ModName`, possibly with different versions?